[docs] Update README (#10472)
expo/expo: docs/README.md

You can find the source of the documentation inside the `pages/versions` directory. Documentation is mostly written in markdown with the help of some React components (for Snack embeds, etc). The routes and navbar are automatically inferred from the directory structure within `versions`.

### Editing Code

> Docs are made in Next.js and TypeScript. If you need to make code changes, follow steps from [Running locally](#running-locally) section, then open a separate terminal and run TypeScript compiler in watch mode - it will watch your code changes and notify you about errors.

Reviewer: ```suggestion
The docs are written with Next.js and TypeScript. If you need to make code changes, follow steps from [Running locally](#running-locally) section, then open a separate terminal and run TypeScript compiler in watch mode - it will watch your code changes and notify you about errors.
```

> The docs are written with Next.js and TypeScript. If you need to make code changes, follow steps from [Running locally](#running-locally) section, then open a separate terminal and run TypeScript compiler in watch mode - it will watch your code changes and notify you about errors.

Reviewer 2: ```suggestion
The docs are written with Next.js and TypeScript. If you need to make code changes, follow steps from the [Running locally](#running-locally) section, then open a separate terminal and run TypeScript compiler in watch mode - it will watch your code changes and notify you about errors.
```

> ```sh

Reviewer 2: ```suggestion

```sh
```

yarn watch
```

When you are done, you should run _prettier_ to format your code. Also, don't forget to run tests and linter before committing your changes.

> ```sh

Reviewer 2: ```suggestion

```sh
```
